src/test_driver/openiotsdk/integration-tests/common/utils.py

```diff
@@ -228,8 +229,11 @@ def send_zcl_command(devCtrl, line):
         if command == None:
             raise exceptions.UnknownCommand(cluster, command)
 
-        err, res = devCtrl.ZCLSend(cluster, command, int(
-                    nodeId), int(endpoint), 0, FormatZCLArguments(cluster, cmdArgsLine, cmdArgsWithType))
+        args = FormatZCLArguments(cluster, cmdArgsLine, cmdArgsWithType)
+        req = eval(
```

Comment:
Can we use `getattr` (https://docs.python.org/3/library/functions.html#getattr) instead of eval ?

Reply:
I'm not sure who should look like then. We need to create the `ClusterObjects.ClusterCommand` class object with specific arguments. Then we pass it to `SendCommand()`.

Reply:
Without error handling:

```python
cluster = getattr(GeneratedObjects, cluster)
command = getattr(cluster.Commands, command)
req = command(**args)
```

I would prefer to avoid `eval` if possible.

Reply:
It works. I applied your suggestion.